deployment/RS03ASHS_Deploy.csv

```diff
@@ -1,13 +1,16 @@
 CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
-TN-313,,,,RS03ASHS-MJ03B-00-OSMOIA301,1,1,2014-09-28T04:02:00,2015-07-08T00:00:00,ATAPL-65244-820-0033,,ATAPL-58333-00003,45.93363,-130.01368,,1541,1541,
+TN-299,,,,RS03ASHS-PN03B-06-CAMHDA301,1,1,2013-08-01T00:00:00,2015-07-08T00:00:00,ATSOO-70232-01H3B,,ATAPL-70168-00001,45.93362,-130.01364,,1542,1542,
```

Comment:
Since deployments for cabled instruments are by instrument (not mooring), maybe it makes more sense to sort the deployments by instrument-deployment (instead of time-instrument) so that we don't mix up deployment numbers for an instrument.